CONTRIBUTING.md

```diff
@@ -10,10 +10,13 @@ If you'd like to contribute, try contributing new symbols or functions that
 KaTeX doesn't currently support. The wiki has a page which lists [all of the
 supported
 functions](https://github.com/Khan/KaTeX/wiki/Function-Support-in-KaTeX) as 
-well as a page that describes how to [examine TeX commands and where to find
-rules](https://github.com/Khan/KaTeX/wiki/Examining-TeX) which can be quite
-useful when adding new commands. You
-can check there to see if we don't support a function you like, or try your
+well as a page that describes how to [examine TeX commands and where to find 
+rules](https://github.com/Khan/KaTeX/wiki/Examining-TeX) which can be quite 
+useful when adding new commands. There's also a user-contributed [preview page]
+(http://utensil-site.github.io/available-in-katex/)
+showing how would KaTex render a series of symbols/functions(including the ones
```

Comment:
I get the "showing how KaTeX would" part, but what does "symbols/functions (including" part mean? I've checked it letter by letter, failed to find the difference.